### DISCUSSION TITLE
Data Links

### DISCUSSION THREAD

**Discussion by jatsqi** [marcusolsson/grafana-treemap-panel#22] [Q&A] [answered]
Hey,

I have been using your plugin for a while now and it is really great!
But is it possible at the moment to use Grafana Data Links with this plugin?
I tried a few things but I didn't manage to get it working.
My goal is to enable the user to click on one of the squares and redirect him to another dashboard.

**Comment by marcusolsson** [comment, marked as answer]
Just tested it. Seems like data link support is broken in Grafana 8. I've created an issue for this: https://github.com/marcusolsson/grafana-treemap-panel/issues/24

Thanks for letting me know!

**Reply by jatsqi** [reply]
Thank you for the fix!
